File: codewars/7-kyu/Building Spheres -> OOP, FUNDAMENTALS/python/solution1.py
from math import pi 

class Sphere:
    def __init__(self, radius, mass):  
        self._r = radius  
        self._m = mass  
    
    def get_radius(self): 
        return self._r
    
    def get_mass(self):
        return self._m
    
    def get_volume(self):
        x = ((4 * pi * (self._r**3)) / 3)
        return round(x, 5)
    
    def get_surface_area(self):
        x = 4 * pi * (self._r**2)
        return round(x, 5)
    
    def get_density(self):
        return round(self.get_mass() / self.get_volume(), 5)

File: codewars/7-kyu/building_spheres-oop-fundamentals/csharp/solution1.cs
using System;


public class Sphere
{
    private int radius;
    private int mass;

    public Sphere(int radius, int mass)
    {
        this.radius = radius;
        this.mass = mass;
    }

    public int GetRadius()
    {
        return radius; 
    }

    public int GetMass()
    {
        return mass; 
    }

    public double GetVolume()
    {
        double x = ((4 * Math.PI * Math.Pow(radius, 3)) / 3);
        return Math.Round(x, 5);
    }

    public double GetSurfaceArea()
    {
        double x = (4 * Math.PI * Math.Pow(radius, 2));
        return Math.Round(x, 5);
    }
  
    public double GetDensity()
    {
        return Math.Round(GetMass() / GetVolume(), 5); 
    }
  
}
